auth0_flutter/ios/Classes/WebAuth/WebAuthModels.swift

```diff
@@ -0,0 +1,15 @@
+enum SafariViewControllerProperty: String, CaseIterable {
+    case presentationStyle
+}
+
+struct SafariViewController {
+    var presentationStyle: UIModalPresentationStyle?
+    
+    static let key = "safariViewController";
+    
+    init(from dictionary: [String: Any?]) {
+        if let presentationStyle = dictionary[SafariViewControllerProperty.presentationStyle] as? Int {
+           self.presentationStyle = UIModalPresentationStyle.init(rawValue: presentationStyle)
```

Comment:
```suggestion
           self.presentationStyle = UIModalPresentationStyle(rawValue: presentationStyle)
```
Calling `init` explicitly is not necessary.